.github/workflows/actions.yml

```diff
@@ -0,0 +1,12 @@
+name: lodge-actions
+
+on: [push]
+
+jobs:
+  ci-pipeline:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: make build
```

Comment:
it should not makes more sense to use paralell jobs for lint and tests?

Reply:
I'm a noob on github actions, I focused on making it work at first! But you are right. This can be turned into a issue and solved later!

Reply:
issue created!